src/test/java/server/request/RequestParserTest.java

```diff
@@ -59,4 +61,14 @@ public void throwsRequestParseExceptionWhenRequestLineIsMalformed() {
             assertEquals(RequestParseException.class, e.getClass());
         }
     }
```

Comment:
Glad you're checking what happens in the case of a malformed request. Seems like there could be a huge number of different bad requests though. Have you thought about what the server would do in some other of those cases?

Reply:
I hadn't, but those would be some good tests to write 😄